public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -786,51 +785,86 @@ public SurveyQuestionResult apply(AchillesResult o) {
         List<DomainInfo> domainInfos =  ImmutableList.copyOf(domainInfoDao.findDomainTotals(getTests, getOrders));
         List<SurveyModule> surveyModules = ImmutableList.copyOf(surveyModuleDao.findByCanShowNotOrderByOrderNumberAsc(0));
 
+        if (domainInfos == null || surveyModules == null) {
```

Comment:
not possible since `surveyModuleDao.findByCanShowNotOrderByOrderNumberAsc(0)` will never return null, only empty list.

Reply:
Same as above this is for the case when domain info and survey module tables are not present.

Reply:
The dao calls will never return null.. only empty list.